courses/beginner/10.md

```diff
@@ -12,30 +12,23 @@ In addition to its own controls, an InSpec profile can bring in the controls fro
 
 When a profile includes controls from another profile, it is usually referred to as a “meta profile” or a “profile overlay.” Those of us with Chef background sometimes call it a “wrapper profile”.
```

Comment:
Those of us with Chef background sometimes call it a “wrapper profile”. ->
Sometimes this is called a “wrapper profile”.

Reply:
Reworded using Eugene's language